### PR TITLE
Add recipes for Ubuntu 21.04 "Hirsute Hippo" and Fedora 34

### PR DESCRIPTION
**Pull request by thaJeztah**

### CODE REVIEW

File: rpm/fedora-34/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,25 @@
+ARG GO_IMAGE
+ARG DISTRO=fedora
+ARG SUITE=33
```
Comment (silvin-lubecki) [inline rpm/fedora-34/Dockerfile:3]: Shouldn't we bump this `SUITE` to `34`? 🤔

Comment (thaJeztah) [reply rpm/fedora-34/Dockerfile:3]: ☺️ 🤦 it helps to actually `git add` after updating 😂 

```bash
git status
On branch add_new_ubuntu_fedora
Your branch is up to date with 'origin/add_new_ubuntu_fedora'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   rpm/fedora-34/Dockerfile
```

Comment (thaJeztah) [reply rpm/fedora-34/Dockerfile]: Pushed 👍

Comment (silvin-lubecki) [reply rpm/fedora-34/Dockerfile]: Thanks 👍

**Review by silvin-lubecki — APPROVED** [review]
LGTM

**Comment by tiborvass** [comment]
This one forgot to update deb/Makefile and rpm/Makefile. See https://github.com/docker/docker-ce-packaging/pull/539

**Pull request merged**